keyboards/skyloong/gk68/pro/config.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+
+#define DRIVER_COUNT 1
+#define DRIVER_1_EN B5
+#define DRIVER_1_CS B12
+#define RGB_MATRIX_LED_COUNT 70
+#define CAPS_LOCK_INDEX 30
+#define WIN_MOD_INDEX 17
+#define MAC_MOD_INDEX 18
+#define SDB B5
+
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
```

Comment:
RGB_MATRIX_KEYPRESSES and FRAMEBUFFER_EFFECTS enabled when `rgb_matrix.animations`, in `info.json`, is defined
```suggestion
#define DRIVER_COUNT 1
#define DRIVER_1_EN B5
#define DRIVER_1_CS B12
#define RGB_MATRIX_LED_COUNT 70
#define CAPS_LOCK_INDEX 30
#define WIN_MOD_INDEX 17
#define MAC_MOD_INDEX 18
#define SDB B5

#define RGB_DISABLE_WHEN_USB_SUSPENDED
```